smslibrary/src/main/java/com/eis/communication/Message.java

```diff
@@ -4,21 +4,40 @@
 
 /**
  * Encapsulation of a message in the network
```

Comment:
*abstraction or what ever you want

Reply:
I don't think you need to review lines with a white background, because that means those are not my changes. I'll fix this anyway because it's wrong.

Reply:
i know
i start reviewing all but I continued reviewing only your adds